src/SPC/command/DownloadCommand.php

```diff
@@ -248,9 +248,9 @@ public function handle(): int
                         $alt_sources = Config::getSource($source)['alt'] ?? null;
                         if ($alt_sources === null) {
                             logger()->warning("No alternative sources found for {$source}, using default alternative source");
-                            $alt_config = array_merge($config, $this->getDefaultAlternativeSource($source));
+                            $alt_config = array_merge($config, Downloader::getDefaultAlternativeSource($source));
                         } elseif ($alt_sources === false) {
-                            logger()->warning("No alternative sources found for {$source}, skipping alternative download");
+                            logger()->error("No alternative sources found for {$source}, skipping alternative download");
```

Comment:
If it's an error, it should not say "skipping alternative download", but throw an appropriate exception text instead.